koku/masu/database/report_manifest_db_accessor.py

```diff
@@ -146,3 +146,39 @@ def get_last_seen_manifest_ids(self, bill_date):
             if not processed:
                 assembly_ids.append(manifest.assembly_id)
         return assembly_ids
+
+    def delete_cost_usage_reports_older_than(self, provider_type, billing_period_start_datetime):
```

Comment:
So, I debated on leaving this comment for a while, but on second thought I don't think the name of your functions accurately portraying what your code is doing here. 

You are not actually deleting the cost usage report data here, you are deleting the cost usage report manifest instead. I am totally open to better names for these, but my suggestion is:

```
def purge_expired_report_manifest(self, provider_type, billing_period_start_datetime):
```
I think this will match the naming format of the function that is actually deleting the cost usage reports `self._cleaner.purge_expired_report_data`

Reply:
sounds good, I plan fix the unit tests first, then re-naming this.

Reply:
@myersCody I renamed it.

Reply:
@nbon12 I like the name that Cody suggested better (`purge_expired_report_manifest`). I feel that it matches the pre-existing pattern and clearly explains what the function is doing. I also feel like the parameter `expired_date` was a good suggestion.

Reply:
@abaiken The pre-existing pattern among other db_accessors is that they only have `get` methods and CRUD operations, I don't see any business logic in the db_accessors, and it seems to be more of a middleware layer that only has knowledge of how to interact with the database. The DB_accessor layer is meant to replace the Django ORM syntax, not to contain any business logic. Isn't the db_accessor middleware layer not supposed to have a concept of the business logic of what's expired and what's not expired? All of the other db_accessors only have `get`, and `delete`, and `update`. Do you want the db_accessor to include the non-CRUD terminology 'expired'? @adberglund

Reply:
@abaiken I renamed it to purge_expired_report_manifest and expired_date